cmd/aries-js-worker/aries.js

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+// TODO not all browsers support private members of classes
+/* @class Aries provides Aries SSI-agent functions. */
+const Aries = new function() {
+    // TODO synchronize access on this map?
+    this._pending = new Map()
+    this._WORKER = new Worker('aries-worker.js')
```

Comment:
what does the all caps convention represent?

Reply:
Sorry, I was not consistent. What is the proper convention?

Reply:
I think camel case would be fine. I think `_` prefix for private fields is a good convention (while browser support for true private fields is lacking).

Reply:
done